### PR TITLE
update recipient model

### PR DESCRIPTION
**Pull request by machi1990**
##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: platform/server/src/resolvers/custom-resolvers.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default {
+    Location: {
```
Comment (machi1990) [inline platform/server/src/resolvers/custom-resolvers.ts:2]: This is needed as ApolloServer seems to be complaining about Location type missing a `__resolveType`

a.k.a without this I had `Type "Location" is missing a "__resolveType" resolver interface GraphQl`. 

Can graphback take care of that instead?

@wtrocki @craicoverflow

Comment (wtrocki) [reply platform/server/src/resolvers/custom-resolvers.ts:2]: There is flag to disable resolve type checks that we use in templates

Comment (machi1990) [reply platform/server/src/resolvers/custom-resolvers.ts:2]: Okay, how do we do that?

File: platform/model/main.graphql [hunk 2]
```diff
@@ -81,6 +83,11 @@ type VolunteerAction {
   distributionCentre: DistributionCentre
 }
 
+interface Location {
```
Comment (wtrocki) [inline platform/model/main.graphql:86]: My recommendation will be to change it to Address and also add 

- geo data lat long
- city
- postcode (optional)

File: platform/server/src/resolvers/custom-resolvers.ts (hunk 1 above)
Comment (wtrocki) [reply platform/server/src/resolvers/custom-resolvers.ts:2]:
```
const schema = makeExecutableSchema({
  typeDefs,
  resolvers,
  resolverValidationOptions: { requireResolversForResolveType: false },
});
```

File: platform/model/main.graphql (hunk 2 above)
Comment (machi1990) [reply platform/model/main.graphql:86]: Let's change it to `Address` and add `city` and `postcode` for now, `lat` and `long` can be added later once we able to retrieve them from map component.

File: platform/server/src/resolvers/custom-resolvers.ts (hunk 1 above)
Comment (machi1990) [reply platform/server/src/resolvers/custom-resolvers.ts:2]: Yes, that could also too. In fact, it was among the explored options before I went with the current option. 

With what you are proposing, it seems like one has to do "manually" while what I was asking is if `graphback` can automatically generate the part I added (also if it seems reasonable?).

**Pull request merged**